Makefile.am

```diff
@@ -13,7 +13,7 @@
 EXTRA_DIST =
 ACLOCAL_AMFLAGS = -I config/m4
 
-noinst_HEADERS = src/uct/api/uct.h src/uct/api/uct_def.h src/uct/api/tl.h
+noinst_HEADERS = src/uct/api/uct.h src/uct/api/v2/uct.h src/uct/api/uct_def.h src/uct/api/tl.h
```

Comment:
pls arrange each H file in a separate line with `\`
see SUBDIRS below as example